tests/strict-equal.test.js

```diff
@@ -0,0 +1,30 @@
+const describe = require('mocha').describe
```

Comment:
I realize the slides have it this way. Just noting that it and describe do not have to be explicitly declared. The way the mocha library is setup, it introduces describe, it and several other functions into our code.